gravitee-apim-cypress/cypress/support/common/api.commands.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { stopApi, deleteApi } from '@commands/management/api-management-commands';
+import { closePlan } from '@commands/management/api-plan-management-commands';
+import { ADMIN_USER } from '@fakers/users/users';
+import { ApiImport } from '@model/api-imports';
+
+declare global {
+  namespace Cypress {
+    interface Chainable<Subject> {
+      removeApi(api: ApiImport): void;
+    }
+  }
+}
+export {};
+
+Cypress.Commands.add('removeApi', (api) => {
```

Comment:
Great idea, it'll simplify test case cleanup.

Some suggestions about this :

- it's hard to make the difference between a call to `deleteApi` and a call to `removeApi`. Maybe we could rename it to be more explicit on what it does ? (but I don't have any good suggestion yet, maybe `purgeApi`?)

- we could make it more generic by looping on all plans to close them all. Like that, it can be used in all test cases with no plan / or more than 1 plan

- Should we use it in other test cases ? Just for example, the `afterEach` of `plans-test` could also use this method.

Reply:
I really like the idea with the loop on all plans! 👍  
So far, there is always only one plan but I'll keep that in mind and will extend the function as soon as we start working on multiple plans.